topic/machine-learning/llm-langchain/cratedb_rag_customer_support_langchain.ipynb

```diff
@@ -410,11 +460,160 @@
     "id": "aQnmpCIZa13L",
     "outputId": "a10c71a7-a6c7-4f83-c069-df49703d4654"
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'To update the shipping address on an existing Microsoft Store order, you can contact Microsoft support either by calling them or reaching out through email. They will assist you in updating the shipping address for your order.'"
+      ]
+     },
+     "execution_count": 48,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "chat_completion.choices[0].message.content"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## CrateDB &#129505; OpenSource\n",
+    "\n",
+    "In the second part of this notebook we build a Retrieval-Augmented Generation (RAG) system using CrateDB with Jina AI and the Llama open-source models. \n",
```

Comment:
We'd suggest to split that into a separate notebook, see our comments above. However, if you think it is right that way, we will accept your decision.

Reply:
Check my comment above. For the presentation purpose it is important to have them together - to avoid redundancy related to data processing step.

Reply:
✅